src/app/data/routes.js

```diff
@@ -18,15 +17,15 @@ import Settings from '../pages/settings';
 import Staging from '../pages/staging';
 import Help from '../pages/help';
 import { getMarketplaceSubnavRoutes } from '../../../vendor/newfold-labs/wp-module-marketplace/components/marketplaceSubnav';
```

Comment:
For these types of imports, looks like we'll need to add `composer install` to the workflow so that the modules are installed and can be found.

Reply:
Should we configure an alias in a `jsconfig.js` file? Would let us a an `import { foo } from @composer/bar/baz` rather than this triple `../`.

Reply:
I like this idea, and I'll create a new PR for it.